src/api/queries/ociQuery.ts

```diff
@@ -0,0 +1,42 @@
+import * as utils from './query';
+
+export interface OciFilters extends utils.Filters {
+  subscription_guid?: string | number;
+}
+
+type OciGroupByValue = string | string[];
+
+interface OciGroupBys {
+  service_name?: OciGroupByValue;
+  subscription_guid?: OciGroupByValue;
+  resource_location?: OciGroupByValue;
+}
+
+interface OciOrderBys {
+  subscription_guid?: string;
+  resource_location?: string;
+  service_name?: string;
+  cost?: string;
+  usage?: string;
+}
```

Comment:
We need to update these properties for OCI.

subscription_guids' > payer_tenant_ids
service_names > product_services
resource_locations > regions